.tekton/scanner-component-pipeline.yaml

```diff
@@ -9,7 +9,7 @@ spec:
   - name: show-sbom
     params:
     - name: IMAGE_URL
-      value: $(tasks.build-container.results.IMAGE_URL)
+      value: $(tasks.build-image-manifest.results.IMAGE_URL)
```

Comment:
Folks here use amd64 image. Not sure if we should too
https://github.com/konflux-ci/build-trusted-artifacts/blob/11bc05e3a30b3870f15b881240515a652cb1e18a/.tekton/build-pipeline.yaml#L10